test/unit/TestGithubUtils.js

```diff
@@ -20,14 +20,14 @@ describe("Test GitHubUtils Clone ", function() {
                     shelljs.rm("-rf", tmpDir);
                 });
         });
-    });
-    
-    //TODO rewrite to ignore timeout on error. this test has been problematic
-//     it("Fails when repo url is not correct", () => {
-//         return gitHubUtils.cloneGitRepo("https://github.com/intuit/saloon.gitXXX", tmpDir)
-//             .catch((response) => {
-//                 expect(response).to.have.string("Error cloning repository");
-//             });
-//     });
+    }).timeout(5000);
 
-});
+    it("Fails when repo url is not correct", () => {
+        setTimeout(() => {}, 2000);
```

Comment:
It may be good to add a log here.
up in the file, define logger:
let logger = require("bunyan");
let log = logger.createLogger({ name: "std-gov-oos" });

```suggestion
        setTimeout(() => {log.info('Ending negative git url clone on timeout');}, 2000);
```